src/utils/formatting.ts

```diff
@@ -99,3 +99,18 @@ export const formatTimeToHMS = (time: string) => {
     seconds
   )}s`;
 };
+
+export const formatMarketId = (marketId: string) => {
+  const timestamp = marketId.slice(0, 6);
+  const location = marketId.slice(6, 9);
+  const place = marketId.slice(9, 11);
+  const date = dayjs(0)
+    .add(+timestamp, "days")
+    .format("DD-MM-YYYY");
+  return `${date} ${location} Race ${place}`;
```

Comment:
we could return all three variables in an object and call the function `getMarketDetails` like we have in the backend - it would help for unit testing